draft-ietf-quic-multipath.md

```diff
@@ -94,35 +94,50 @@ Connection migration as specified in {{Section 9 of QUIC-TRANSPORT}}
 directs a peer to switch sending through
 a new preferred path, and, if successful, to release resources
 associated with the old path. The multipath extension specified in this document
-builds on this mechanism but introduces a path identifier (see {{explicit-path-id}})
-to manage connection IDs and packet number spaces per path (see {{num-spaces}}).
+builds on this mechanism but introduces a path identifier (Path ID)
+to manage connection IDs and packet number spaces per path, enabling the use
+of multiple paths simultaneously.
+
+The connection ID of a packet binds the packet to a path identifier, and therefore
+to a packet number space. That means each connection ID is associated with exactly one path identifier
+but multiple connection IDs are usually issued for each path identifier.
+The same Path ID is used in both directions, starting with 0 for the initial path.
+Path identifies are generated monotonically increasing and cannot be reused.
+
+This extension uses multiple packet number spaces, one for each path.
+Each Path ID-specific packet number space starts at packet number 0.
+As such, each path maintains distinct packet number states for sending and receiving packets, as in {{QUIC-TRANSPORT}}.
+Using multiple packet number spaces enables direct use of the
+loss detection and congestion control mechanisms defined in
+{{QUIC-RECOVERY}} on a per-path basis.
+However, use of multiple packet number spaces requires
+non-zero connection IDs in order to identify the path and the respective
+packet number space as well as a modified AEAD calculation including the
+Path ID (see {{nonce}}).
```

Comment:
I removed the subsections in the intro and moved these two paragraphs here because of work on section 2 there were only one paragraph left in each of the subsection.